src/modules/system/data/CronOptions.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <string>
+#include <vector>
+
+namespace faker
+{
+struct CronOptions {
+    bool includeYear = false;
+    bool includeNonStandard = false;
+};
+
+const std::vector<std::string> CRON_DAY_OF_WEEK = {"SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT"};
```

Comment:
Please use camelCase for all types of variables

Reply:
Done